app/views/orders/search.html.erb

```diff
@@ -23,9 +22,9 @@
           <% @orders.each_with_index do |order| %>
             <tr>
               <td><%= link_to order.state, order_path(order.guid) %></td>
-              <td><%= DateTime.parse(order.created_date).to_s %></td>
-              <td><%= DateTime.parse(order.submitted_date).to_s %></td>
-              <td><%= DateTime.parse(order.updated_date).to_s %></td>
+              <td><%= safe_date_output order.created_date, "Not Created" %></td>
```

Comment:
you've mixed single and double quotes on these three lines

Reply:
will do